src/components/Theme/src/default-theme.js

```diff
@@ -15,6 +15,21 @@ export default function defaultTheme() {
 			elevation: '#ffffff',
 			overlay: 'rgba(0, 0, 0, 0.3)',
 			primary: '#000000',
+			critical: {
+				fill: '#cd2026',
+				text: '#a82826',
+				subtle: '#f6eceb',
```

Comment:
We should create more documentation around these colors and how they are used. I understand text, but the term fill and subtle are less common names.

It might be good to get aligned with design on nomenclature and consider how we're documenting the terms and values for both design and engineering. I can't find a reference in our design documentation.

Reply:
already reached alignment with Design on these, these are the terms Design uses